pkg/services/sqlstore/annotation_test.go

```diff
@@ -14,275 +14,241 @@ func TestAnnotations(t *testing.T) {
 	mockTimeNow()
 	defer resetTimeNow()
 	InitTestDB(t)
+	repo := SqlAnnotationRepo{}
+
+	t.Run("Testing annotation create, read, update and delete", func(t *testing.T) {
+		t.Cleanup(func() {
+			_, err := x.Exec("DELETE FROM annotation WHERE 1=1")
+			assert.Nil(t, err)
+			_, err = x.Exec("DELETE FROM annotation_tag WHERE 1=1")
+			assert.Nil(t, err)
+		})
 
-	Convey("Testing annotation saving/loading", t, func() {
-		repo := SqlAnnotationRepo{}
-
-		Convey("Can save annotation", func() {
-			Reset(func() {
-				_, err := x.Exec("DELETE FROM annotation WHERE 1=1")
-				So(err, ShouldBeNil)
-				_, err = x.Exec("DELETE FROM annotation_tag WHERE 1=1")
-				So(err, ShouldBeNil)
-			})
-
-			annotation := &annotations.Item{
+		annotation := &annotations.Item{
+			OrgId:       1,
+			UserId:      1,
+			DashboardId: 1,
+			Text:        "hello",
+			Type:        "alert",
+			Epoch:       10,
+			Tags:        []string{"outage", "error", "type:outage", "server:server-1"},
+		}
+		err := repo.Save(annotation)
+		assert.Nil(t, err)
```

Comment:
You ought to use `require` when checking errors, since `assert` will continue execution, which leads to chaos after errors occur:
```suggestion
		require.Nil(t, err)
```

Reply:
Great feedback, thanks.